moskito-frontend/src/hooks/useCampsites.js

```diff
@@ -1,21 +1,36 @@
 import {useState} from 'react'
 import { setFeatureFilter, getTrueFilter }from '../services/filterServices'
+import {  getFetch } from '../lib/fetch'
+import { loadFromLocal } from '../lib/localStorage'
+
+const token = loadFromLocal('tokens')
 
 
 export default function useCampsites() {
 
     const [ campsites, setCampsites ] = useState([])
+    const [ bookmarks, setBookmarks ] = useState([])
    
 
     function getCampsites(filter) {
         const trueFilter = getTrueFilter(filter)
-            setFeatureFilter(trueFilter)
-            .then(result => setCampsites(result))
-            .catch(error => {
-                console.log('error', error)
-            })
+        setFeatureFilter(trueFilter)
+        .then(result => setCampsites(result))
+        .catch(error => {
+            console.log('error', error)
+        })
       }
+      
+    function getBookmarks(){
+        const baseUrl = "http://moskito.local/bookmark"
+        const myHeaders = new Headers();
+        myHeaders.append("Authorization", `Bearer ${token.value}`);
```

Comment:
You could just pass an object literal instead of using the constructor here. That would fit the functional programming style a bit better.

Example:
```js
const headers = {
"Authorization": `Bearer ${token.value}`
}
```
and then `getFetch( headers, baseUrl )` or myHeaders if you like :)